rapidtide/workflows/rapidtide.py

```diff
@@ -2701,13 +2701,21 @@ def rapidtide_main(argparsingfunc):
         del fitcoeff
         del fitNorm
 
-    for mapname, mapsuffix in [("meanvalue", "mean")]:
+    temp = {
+        "meanvalue": {
+            "data": meanvalue,
+            "suffix": "mean",
+        }
+    }
+    for mapname, temp_dict in temp.items():
+        outmaparray = temp_dict["data"]
+        mapsuffix = temp_dict["suffix"]
+
```

Comment:
Must revert. Was related to an invalid combination of arguments that slipped through and resulted in `meanvalue` not being defined. This change was just part of my debugging and I accidentally committed it.